scripts/dotnet.py

```diff
@@ -621,9 +621,10 @@ def get_commit_date(
         url = urlformat % (owner, repo, commit_sha)
 
     build_timestamp = None
+    item = None
     retrycount = 0
     success = 0
-    while success == 0 and retrycount <= 3:
+    while success == 0 and retrycount <= 5:
```

Comment:
Not really caused by the changes in this PR, but this should be replace-able with a for loop over range(6) and using a `break` instead of the `success` variable.